modules/api/src/main/java/org/apache/fluo/api/client/scanner/ScannerBuilder.java

```diff
@@ -30,6 +31,96 @@
    */
   ScannerBuilder over(Span span);
 
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#exact(Bytes)
+  * @param row restrict the scanner to data in an exact row
+  * @return self
+  */
+  default ScannerBuilder over(Bytes row) {
+    return over(Span.exact(row));
+  }
+
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#exact(CharSequence)
+  * @param row restrict the scanner to data in an exact row. String parameters will be encoded as UTF-8
+  * @return self
+  */
+  default ScannerBuilder over(CharSequence row) {
+    return over(Span.exact(row));
+  }
+
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#exact(Bytes, Column)
+  * @param row restrict the scanner to data in an exact row 
+  * @param col restrict the scanner to data in exact {@link org.apache.fluo.api.data.Column}.
+  */
+  default ScannerBuilder over(Bytes row, Column col) {
+    return over(Span.exact(row, col));
+  }
+
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#exact(CharSequence, Column)
+  * @param row restrict the scanner to data in an exact row
+  * @param col restrict the scanner to data in exact {@link org.apache.fluo.api.data.Column}. 
+  * @return self
+  */
+  default ScannerBuilder over(CharSequence row, Column col) {
+    return over(Span.exact(row, col));
+  }
+
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#prefix(Bytes)
+  * @param rowPrefix restrict the scanner to data in rows that begins with a prefix
+  * @return self
+  */
+  default ScannerBuilder overPrefix(Bytes rowPrefix) {
+    return over(Span.prefix(rowPrefix));
+  }
+
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#prefix(CharSequence)
+  * @param rowPrefix restrict the scanner to data in rows that begins with a prefix.
+  * @return self
+  */
+  default ScannerBuilder overPrefix(CharSequence rowPrefix) {
+    return over(Span.prefix(rowPrefix));
+  }
+
+  /**
+  *
+  * @since 1.2.0
+  * @see org.apache.fluo.api.data.Span#prefix(Bytes, Column)
+  * @param rowPrefix restrict the scanner to data in rows that begins with a prefix 
```

Comment:
This method scans a column prefix within an exact row.  So, need to change `rowPrefix` to `row`.   Also the javadoc should say something like `restrict the scanner to data in specified row`.